tools/rebase-onto-formatter.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "INSTRUCTIONS:"
+echo "1. Make a copy of your branch (or be prepared to salvage it from reflog)."
+echo "2. Find out the what the base commit is (in our case ...)"
```

Comment:
```suggestion
echo "2. Find out what the base commit is (in our case...); consider using git merge-base for this."
```

(not sure what the `in our case` refers to.)